ts/src/ndax.ts

```diff
@@ -1334,7 +1338,16 @@ export default class ndax extends Exchange {
         const defaultAccountId = this.safeInteger2 (this.options, 'accountId', 'AccountId', parseInt (this.accounts[0]['id']));
         const accountId = this.safeInteger2 (params, 'accountId', 'AccountId', defaultAccountId);
         const clientOrderId = this.safeInteger2 (params, 'ClientOrderId', 'clientOrderId');
-        params = this.omit (params, [ 'accountId', 'AccountId', 'clientOrderId', 'ClientOrderId' ]);
+        let orderType = this.safeInteger (this.options['orderTypes'], this.capitalize (type));
```

Comment:
@samgermain I don't love this logic because it does not allow the user to use the exchange-specific type if needed

Reply:
They still can, they just need to set `params['OrderType']`

Reply:
@samgermain yeah but I meant using the `type` parameter

Reply:
This is how it was set up before, but I added the string numbers as keys to options['orderTypes'] because the method expects a string for type. We can't really do any better than that without changing the type of `type` to `string | number` across all exchanges

Reply:
yes accepting '1' is ok